src/utils/usageReportConsent.js

```diff
@@ -0,0 +1,18 @@
+const KEY = 'usage-data-consent'
+const confirmKey = 'usage-data-consent-confirmed-once'
```

Comment:
If you want to keep this, consider using the same casing for both keys.

You could also consider using a single key (`usage-data-consent`) with the following values:
`true`: the user accepts
`false`: the user refuses
`null or undefined`: the user has not specified any preference yet
I would still keep `getConsentConfirmationStatus()` to check for null/undefined for proper encapsulation.